pallets/api/src/nonfungibles/mod.rs

```diff
@@ -0,0 +1,595 @@
+//! The non-fungibles pallet offers a streamlined interface for interacting with non-fungible
+//! assets. The goal is to provide a simplified, consistent API that adheres to standards in the
+//! smart contract space.
+
+use frame_support::{
+	dispatch::WithPostDispatchInfo,
+	traits::{nonfungibles_v2::Inspect, Currency},
+};
+use frame_system::pallet_prelude::BlockNumberFor;
+pub use pallet::*;
+use pallet_nfts::WeightInfo as NftsWeightInfoTrait;
+pub use pallet_nfts::{
```

Comment:
Not sure whether this `pub` is necessary:
```suggestion
pub use pallet_nfts::{
```

Reply:
Those types will be used to implement the pop-api for non-fungibles and integration tests. Even though they can be imported from pallet_nfts, making this public to avoid adding `pallet-nfts` crate.